README.md

```diff
@@ -23,11 +26,13 @@ Options:
           Port to listen for https connection on (will use the interfaces already defined) [env: TLS_SERVER_PORT=] [default: 3043]
 ```
 
+All supported [modes](#modes) share common configuration variables.
```

Comment:
I'm not sure what this means here. Different modes will definitely have different variables, some modes will share some stuff

Reply:
Oooohhh. Yeah sure, the original is not a very clear description to me. I think you're right - let's remove it and put it back in when we actually have that section stabilised. Also okay with just leaving it. Gonna slap an approve on this and leave it up to you